package/yast2.changes

```diff
@@ -1,4 +1,17 @@
 -------------------------------------------------------------------
+Wed Nov  5 09:33:08 UTC 2014 - mfilka@suse.com
+
+- bnc#900383
+  - ported several fixes for device renaming
+  - added net device type detection based on sysfs
+  - fixed type detection workflow
+  - fixed device type detection when commiting new device into
+    NetworkInterfaces' cache.
+  - changes API for device type detection - incompatible to previous
+    versions
```

Comment:
via bugzilla or I can update ```Conflicts``` directive in spec file. The second one seems better to me. Do you agree with the proposal or you have an other idea?